cdc/kv/client.go

```diff
@@ -423,13 +425,11 @@ func (s *eventFeedSession) eventFeed(ctx context.Context) error {
 
 	g, ctx := errgroup.WithContext(ctx)
 
-	g.Go(func() error {
-		return s.dispatchRequest(ctx)
-	})
+	g.Go(func() error { return s.dispatchRequest(ctx) })
```

Comment:
I would prefer still keep the old format, since it's easier for the bugger to show the line number.